libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -1562,6 +1579,69 @@ void AP_OSD_Screen::draw_throttle(uint8_t x, uint8_t y)
     backend->write(x, y, false, "%3d%c", gcs().get_hud_throttle(), SYMBOL(SYM_PCNT));
 }
 
+void AP_OSD_Screen::draw_sidebars(uint8_t x, uint8_t y)
```

Comment:
should add a HAL_OSD_SIDEBAR_ENABLE #if to allow saving the flash space

Reply:
done